components/tekton-ci/base/repository.yaml

```diff
@@ -103,3 +103,10 @@ metadata:
   name: pipeline-service
 spec:
   url: "https://github.com/openshift-pipelines/pipeline-service"
+---
+apiVersion: pipelinesascode.tekton.dev/v1alpha1
+kind: Repository
```

Comment:
Shouldn't we remove old one?

Reply:
by old one if you mean [this repository](https://github.com/redhat-appstudio/infra-deployments/blob/main/components/promotion/base/repository.yaml)  which is in staging cluster, then yes pipeline service team is planning to remove it